installation-bundle/src/Database/Installer.php

```diff
@@ -110,7 +111,7 @@ public function compileCommands(): void
         $fromSchema = $this->connection->createSchemaManager()->createSchema();
         $toSchema = $this->schemaProvider->createSchema();
 
-        $diff = $fromSchema->getMigrateToSql($toSchema, $this->connection->getDatabasePlatform());
+        $diff = (new Comparator())->compareSchemas($fromSchema, $toSchema)->toSql($this->connection->getDatabasePlatform());
```

Comment:
This should take `createComparator()` on the platform if DBAL 3.x in order to fix the wrong lenght comparisons.

Reply:
Changed in 39895e4c1ad86ceedc986e8b559c0d9a186de700.